daisy/workflow.go

```diff
@@ -254,6 +263,11 @@ func (w *Workflow) RunWithModifiers(
 		w.LogWorkflowInfo("Error running workflow: %v", err)
 		return err
 	}
+	if w.outputValues != nil {
+		for k, v := range w.outputValues {
+			w.LogWorkflowInfo("Serial-output value -> %v:%v", k, v)
```

Comment:
I assume, in import/export wrappers, we will read w.outputValues directly and log them to sawmill? If so, this output is just for logging purposes so it's visible in main Daisy log, not only VM serial console output?

If we want to read w.outputValues in wrappers, it should be exported or a getting func provided in workflow.go